extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java

```diff
@@ -214,14 +216,9 @@ void nativeImageConfiguration(
                 "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.11.xsd",
                 "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.12.xsd",
                 "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.13.xsd",
+                "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.14.xsd",
+                "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd",
```

Comment:
So I added this in case users prefer that over an explicitly versioned file.
I'm still not sure whether we should use it in our tests as well. If 4.15 comes out and Quarkus does not yet use it then there might be a mismatch, dunno.

This is also related to this message that liquibase is now printing if using an older version:
> INFO: An older version of the XSD is specified in one or more changelog's <databaseChangeLog> header. This can lead to unexpected outcomes. If a specific XSD is not required, please replace all XSD version references with "-latest". Learn more at https://docs.liquibase.com

Reply:
Yeah, no strong opinion here either